config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -196,7 +196,7 @@ periodics:
       # Skipping "Should recreate evicted statefulset" because of an issue in dockershim for dualstack
       # Suggested fix - https://github.com/kubernetes/kubernetes/pull/94382
       - --test_args=--ginkgo.focus=\[Feature:IPv6DualStackAlphaFeature\]|\[Conformance\] --ginkgo.skip=\[Slow\]|\[Serial\]|\[Flaky\]|Should.recreate.evicted.statefulset
-      - --ginkgo-parallel=20
+      - --ginkgo-parallel=1
```

Comment:
IIRC you just need to remove the ginkgo-parallel line entirely.

In that case you can run the serial tests too, that are being skipped now, FYI

Reply:
So I'm following the same format for all the jobs defined in the yaml: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-network/dualstack-e2e.yaml#L152.

We can enable `Serial` tests in a follow up PR after few test runs. I'm planning to do it incrementally 😄 

btw, `1.20` branch tests are running all the dual-stack tests as expected.

Reply:
fair enough